README.md

```diff
@@ -49,7 +49,7 @@ To use the reference implementation add the following dependency to your project
 ```xml
 <dependency>
     <groupId>io.github.a2asdk</groupId>
-    <artifactId>a2a-java-reference-server</artifactId>
+    <artifactId>a2a-java-sdk-reference-jsonrpc</artifactId>
```

Comment:
It would be good to add something about the `grpc` dep to the README too. That could be addressed in a follow-up issue instead though.